README.md

```diff
@@ -33,16 +33,17 @@ https://github.com/Superflows-AI/superflows/assets/33871096/3cc09611-a358-4a04-b
 ## Features
 
 - [x] Calls API endpoints to complete tasks and answer questions for users
+- [x] Writes code to analyse data and product visualisations 📊
```

Comment:
Smaller, simpler words are easier to read and understand quickly and with less effort
```suggestion
- [x] Writes code to analyse data and create plots 📊
```